codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Settings.java

```diff
@@ -176,7 +183,7 @@ public Http2Settings copyFrom(Http2Settings settings) {
         return this;
     }
 
-    Integer getIntValue(int key) {
+    public Integer getIntValue(int key) {
```

Comment:
@Scottmitch I think this method would be useful, so let me just clarify in the javadocs.